### PR TITLE
update ampproject.org redirects

### LINKED ISSUE #1789: blog post redirects from ampproject.org don't work
## Description of the issue found on ampproject.org

A blog post link clicked from ampproject.org that would redirect to amp.dev goes to the main blog page instead of the specific blog post

## How do we reproduce the issue?

Please provide the steps to reproduce the issue:

1. - Google Search: [Standardizing lessons learned from AMP]
2. Click first link: https://www.ampproject.org/latest/blog/standardizing-lessons-learned-from-amp/
3. Goes to https://blog.amp.dev/

What browser are you using? Chrome
What O/S are you using?

### PR DESCRIPTION
**Pull request by patrickkettner**
fixes #1789 and #2796

For 1789, I crawled all the URLs used on the blog, and the one mentioned is the only one not current redirecting. 
I update the [links mentioned in a comment](https://github.com/ampproject/amp.dev/issues/2796#issuecomment-523112744) for 2796.

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -227,6 +235,11 @@
           "destination": "https://blog.amp.dev/2018/11/30/amp-projects-new-governance-model-now-in-effect/amp/",
           "type": 301
       },
+      {
+          "source": "/latest/blog/ads-amp-roadmap-update-for-end-of-q2",
+          "destination": "https://blog.amp.dev/2017/07/14/amp-roadmap-update-for-end-of-q2/amp/",
```
Comment (westonruter) [inline firebase.json:240]:
```suggestion
          "destination": "https://blog.amp.dev/2017/07/14/amp-roadmap-update-for-end-of-q2/",
```

File: firebase.json [hunk 2]
```diff
@@ -282,6 +295,11 @@
           "destination": "https://blog.amp.dev/2017/11/28/an-amp-paywall-and-subscription-model-for-all-publishers/amp/",
           "type": 301
       },
+      {
+          "source": "/latest/blog/ads-an-amp-paywall-and-subscription-model-for-all-publishers",
+          "destination": "https://blog.amp.dev/2017/11/28/an-amp-paywall-and-subscription-model-for-all-publishers/amp/",
```
Comment (westonruter) [inline firebase.json:300]:
```suggestion
          "destination": "https://blog.amp.dev/2017/11/28/an-amp-paywall-and-subscription-model-for-all-publishers/",
```

File: firebase.json [hunk 3]
```diff
@@ -689,11 +707,21 @@
       "type": 301
       },
       {
+      "source": "/:lang/latest/blog/ads-2058",
+      "destination": "https://blog.amp.dev/2018/06/06/2058/amp/",
```
Comment (westonruter) [inline firebase.json:711]:
```suggestion
      "destination": "https://blog.amp.dev/2018/06/06/2058/",
```

**Comment by patrickkettner** [comment]
@westonruter done!

File: firebase.json [hunk 4]
```diff
@@ -105,7 +105,12 @@
         "source": "/amp-conf-2018",
         "destination": "https://amp.dev/events/amp-conf-2019?referrer=ampproject.org",
         "type": 301
+      }{
```
Comment (matthiasrohmer) [inline firebase.json:108]: Missing comma.

File: firebase.json [hunk 5]
```diff
@@ -140,1081 +145,1371 @@
...
       {
+      "source": "/:lang/latest/blog/ads-child-mind-institute-boosts-social-shares-on-amp-pages-with-addthis",
+      "destination": "https://blog.amp.dev/2018/10/23/child-mind-institute-boosts-social-shares-on-amp-pages-with-addthis/",
+      "type": 301
+      },
       "source": "/:lang/latest/blog/child-mind-institute-boosts-social-shares-on-amp-pages-with-addthis",
-      "destination": "https://blog.amp.dev/2018/10/23/child-mind-institute-boosts-social-shares-on-amp-pages-with-addthis/amp/",
+      "destination": "https://blog.amp.dev/2018/10/23/child-mind-institute-boosts-social-shares-on-amp-pages-with-addthis/",
       "type": 301
       },
```
Comment (matthiasrohmer) [inline firebase.json:947]: GitHub marks some bad chars here - but that shouldn't hurt.

**Comment by patrickkettner** [comment]
@matthiasrohmer the travis build is failing, any idea as to why?

**Comment by patrickkettner** [comment]
@matthiasrohmer it looks like it is failing while compiling an optional build module, any idea?

**Review by sebastianbenz — APPROVED** [review]

**Comment by patrickkettner** [comment]
@matthiasrohmer @sebastianbenz build is working, but deploy itself is failing

```
BucketNotFoundException: 404 gs://ampproject-staging.appspot.com bucket does not exist.
CommandException: 1 file/object could not be transferred.
```


any ideas?

**Comment by sebastianbenz** [comment]
The staging project might've been deleted. You need to deploy to prod.

**Comment by patrickkettner** [comment]
how does one do that?

**Comment by sebastianbenz** [comment]
Merge into `origin/production-legacy`

>

**Pull request merged**